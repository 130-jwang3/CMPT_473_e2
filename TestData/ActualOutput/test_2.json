[{"id":2,"name":"Jane \"the great\""},{"id":3,"name":"Alice"}]