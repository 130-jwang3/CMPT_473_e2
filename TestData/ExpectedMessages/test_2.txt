Converting TestData/TestFiles/test_2.csv -> TestData/ActualOutput/test_2.json [sep=,, header=1]
Only keeping rows: 1 2 3 4
   id              name  age
1   2  Jane "the great"   25
2   3             Alice   40
