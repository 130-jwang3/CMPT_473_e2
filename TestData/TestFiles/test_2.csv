"id","name","age"
"id","name","age"
"id","name","age"
"1","John, Doe","30"
"2","Jane ""the great""","25"
"3","Alice","40"