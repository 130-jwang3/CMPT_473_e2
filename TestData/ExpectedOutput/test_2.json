[{"id":2,"name":"Jane \"the great\"","age":25},{"id":3,"name":"Alice","age":40}]